framework/physics/FieldOperations/PartitionerPredicate.h

```diff
@@ -0,0 +1,31 @@
+#ifndef CHITECH_PARTITIONERPREDICATE_H
+#define CHITECH_PARTITIONERPREDICATE_H
+
+#include "field_operation.h"
+
+namespace chi
+{
+class GraphPartitioner;
+}
+
+namespace chi_physics::field_operations
+{
+
+class PartitionerPredicate : public FieldOperation
```

Comment:
I won't be using this, but I am unsure what "PartitionerPredicate" means. Am I reading the source code correctly when I interpret this as plopping cell processor IDs into a specific component of a specific field function? Is the idea that you could create a mesh and a FF in Lua and then apply this?

Reply:
This might help a little.
```
params.SetGeneralDescription("Field operation that will write to the "
                               "field the result of a partitioning operation");
```
It has two use cases thus far. Diagnostic when partitioning in serial goes wrong, and if you want to build volumetric sub-sets of cells.